### ISSUE TITLE
Cannot connect to neurogitea

### ISSUE THREAD

**Issue by jcohenadad** [neuropoly/data-management#318]
i keep having connection issue and i keep forgetting what's the solution so i'm just going to create an issue so i can quickly go back to it everytime i have that problem--

![image](https://github.com/neuropoly/data-management/assets/2482071/92272a41-a5b9-44be-a90c-c8f40abe3ad6)

**Comment by jcohenadad** [comment]
❌ Does not work:

~~~
julien-macbook:~ $ ssh -N -L 3000:localhost:3000 p101317@data.neuro.polymtl.ca
p101317@data.neuro.polymtl.ca's password: 
Permission denied, please try again.
~~~

**Comment by jcohenadad** [comment]
❌ Does not work:

~~~
julien-macbook:~ $ ssh -N -L 3000:localhost:3000 p101317@GRAMES.polymtl.ca@data.neuro.polymtl.ca
p101317@GRAMES.polymtl.ca@data.neuro.polymtl.ca's password: 
~~~
(keeps hanging)

after reading that it is normal for it to keep hanging, i did what was asked below, but no success

![image](https://github.com/neuropoly/data-management/assets/2482071/e06ea848-64d2-47a9-ba06-8e49f55cc70f)


i tried with p101317 and wth p101317@GRAMES.polymtl.ca

**Comment by namgo** [comment]
@mguaypaq I remember we had discussed creating a local account for Julien - is this still an option?

**Comment by jcohenadad** [comment]
because i keep needing the list of datasets (sometimes very urgently for admin/ethics purposes), until this is resolved, i'm just going to list the datasets below:

<details><summary>Terminal output</summary>

```console
datasets/basel-mp2rage
datasets/bavaria-quebec-spine-ms
datasets/bavaria-quebec-spine-ms-unstitched
datasets/beijing-tumor
datasets/canproco
datasets/data_axondeepseg_bf_source
datasets/data_axondeepseg_bf_training
datasets/data_axondeepseg_cars
datasets/data_axondeepseg_sickkids
datasets/data_axondeepseg_stanford
datasets/data_axondeepseg_tem
datasets/data_axondeepseg_users
datasets/data_axondeepseg_vcu
datasets/data_axondeepseg_wakehealth_source
datasets/data_axondeepseg_wakehealth_training
datasets/dcm-oklahoma
datasets/dcm-zurich
datasets/dcm-zurich-lesions
datasets/dcm-zurich-lesions-20231115
datasets/eeg-epilepsy
datasets/exvivo-spinal-cord
datasets/gmseg-challenge-2016
datasets/goldatlas
datasets/inspired
datasets/levin-stroke
datasets/lumbar-epfl
datasets/lumbar-nusantara
datasets/lumbar-vanderbilt
datasets/marseille-3t-mp2rage
datasets/marseille-rootlets
datasets/marseille-t2s-template
datasets/mni-bmpd
datasets/model_seg_exvivo_gm-wm_t2_unet2d-multichannel-softseg
datasets/mrspineseg-challenge-2021
datasets/msseg_challenge_2016
datasets/msseg_challenge_2021
datasets/nih-ms-mp2rage
datasets/philadelphia-pediatric
datasets/sci-colorado
datasets/sci-paris
datasets/sci-zurich
datasets/sct-testing-large
datasets/sct-testing-release
datasets/spider-challenge-2023
datasets/synthrad-challenge-2023
datasets/template
datasets/template_dog_virginiatech
datasets/twh-rootlets
datasets/uk-biobank
datasets/uk-biobank-processed
datasets/umass-ms-ge-excite1.5
datasets/umass-ms-ge-hdxt1.5
datasets/umass-ms-ge-pioneer3
datasets/umass-ms-siemens-espree1.5
datasets/uqueensland_mouse
datasets/whole-spine
datasets/zurich-mouse
```

</details>

**Comment by nullnik-0** [comment]
> I remember we had discussed creating a local account for Julien - is this still an option?

Yes, I did this a while ago. I'm just trying to find the issue where I left the instructions!

**Comment by nullnik-0** [comment]
[this is the issue](https://github.com/neuropoly/computers/issues/683#issuecomment-2099860868)

@jcohenadad do you still have the credentials that I sent you?

**Comment by jcohenadad** [comment]
> @jcohenadad do you still have the credentials that I sent you?

yes-- sorry, i had forgotten-- i was able to log in with it-- i hope the issue with my username will be fixed at some  point

**Comment by nullnik-0** [comment]
Some updates: 

- It looks like upstream has finally added julien to neuropoly (GE) and removed him from GRAMES.
- This means that using  `p101317@GRAMES.polymtl.ca` _should, in theory_ no longer be necessary with this account.
- Except that I checked the auth logs on data and Julien seems to be authenticating successfully with `p101317@GRAMES.polymtl.ca`, which doesn't make much sense
- @namgo and I talked about this in our meeting, and he is going to look into this more!
- Good news is that [upstream opened port 80 for us](https://github.com/neuropoly/intranet.neuro.polymtl.ca/issues/117#issuecomment-2135560727), so the port forwarding command should no longer be needed, and you should be able to connect to the site as normal while on the VPN.

**Comment by jcohenadad** [comment]
Sorry team, but I still cannot login to neurogitea. Following https://github.com/neuropoly/computers/issues/683#issuecomment-2860575890 I've tried with p101317, and with my GE password, but nothing works. I've tried >10 combinations (I have multiple GE/poly usernames/passwords depending on the service so it's a bit of a mess...), and so maybe my account has been locked

**Comment by nullnik-0** [comment]
@jcohenadad, I'm going to close this issue and reopen [#683 in computers](https://github.com/neuropoly/computers/issues/683#issuecomment-3014221569) instead, as the place to follow up on this, since it has the most recent and up-to-date information about the status of your NeuroGitea account, and I'd like to track this all in one place as much as possible. 

I'm out of hours for the week, but I'll take a look at the issue first thing Monday.
